dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -214,11 +238,29 @@
         <Setter Property="HighContrastAdjustment" Value="None" />
     </Style>
 
+    <Style x:Key="InkToolbarPenConfigurationControlTextStyle" TargetType="TextBlock" BasedOn="{StaticResource BodyTextBlockStyle}">
+        <Setter Property="FontWeight" Value="SemiBold" />
+    </Style>
+    
     <Style x:Key="InkToolbarSelectionAccentStyle" TargetType="Rectangle">
         <Setter Property="Opacity" Value="0" />
         <Setter Property="VerticalAlignment" Value="Bottom" />
-        <Setter Property="Height" Value="2" />
-        <Setter Property="Fill" Value="{ThemeResource InkToolbarAccentColorThemeBrush}" />
+        <Setter Property="Height" Value="3" />
+        <Setter Property="Width" Value="16"/>
+        <Setter Property="Fill" Value="{ThemeResource InkToolbarAccentColorForegroundThemeBrush}" />
+        <contract7Present:Setter Property="RadiusX" Value="1.5" />
+        <contract7Present:Setter Property="RadiusY" Value="8" />
+    </Style>
+
+    <Style x:Key="InkToolbarFlyoutItemSelectionAccentStyle" TargetType="Rectangle">
+        <Setter Property="Opacity" Value="0" />
+        <Setter Property="VerticalAlignment" Value="Center" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+        <Setter Property="Height" Value="16" />
+        <Setter Property="Width" Value="3"/>
+        <Setter Property="Fill" Value="{ThemeResource InkToolbarAccentColorForegroundThemeBrush}" />
+        <contract7Present:Setter Property="RadiusX" Value="8" />
```

Comment:
8 seems awfully high.... shouldn't this be 1.5 too?

Reply:
It is half of Height, else round corner are not getting visible correctly.

Reply:
Being half of height would make it an oval of some kind. That doesn't make sense.